.github/workflows/root-ci.yml

```
@@ -356,6 +356,8 @@ jobs:
             overrides: ["imt=Off", "LLVM_ENABLE_ASSERTIONS=On", "CMAKE_BUILD_TYPE=Debug"]
           - image: ubuntu2310
             overrides: ["LLVM_ENABLE_ASSERTIONS=On", "CMAKE_CXX_STANDARD=20"]
+          - image: ubuntu2404
```

```suggestion
          - image: ubuntu24
```
For the sake of consistency with previous LTS releases in the CI: maybe drop the month and just call it `ubuntu24`? Or was this deliberate?

---

It was deliberate to be clear about the exact version.